lib/datasource/nuget/index.js

```diff
@@ -1,51 +1,190 @@
-const parse = require('github-url-from-git');
```

Comment:
Shouldn't we still use this to parse the URLs?

Reply:
I returned `sourceUrl` verification if https://github.com/renovatebot/renovate/pull/3148/commits/dbb98ecf5c979c41fb4464436ac07dc3170bc6de